wordpress-mobile/WordPress-Android
# Fix notification detail crash for GB File block and similar objects

Fixes #16313

All the code changes are in [this](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2357) Flux-C PR.

To test:
- Open a notification detail of the comment with a GB File block.
- Make sure it is not crashing.

## Regression Notes
1. Potential unintended areas of impact
- None as far as I can tell.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
- Manual and existing client side and Flux-C testing.

3. What automated tests I added (or what prevented me from doing so)
Custom getter test in Flux-C.

PR submission checklist:
